cliquet/resource/__init__.py

```diff
@@ -1046,6 +1088,17 @@ def get_parent_id(self, request):
         """
         return ''
 
+    def _extract_fields(self):
+        """Override default fields extraction from QueryString to allow
+        the permission_field to exist.
+        """
+
+        fields = super(ShareableResource, self)._extract_fields()
+        if fields:
+            fields.append(self.model.permissions_field)
+
+        return fields
```

Comment:
Without this there is no permission displayed when we get a record. There is no reason to hide it when we ask for partial_fields. It feels wrong when I test it and did not receive the permissions.

If you need write permission to display permissions, it should not be done here since it has nothing to do with partial response.